thirdparty/mbedtls/include/godot_core_mbedtls_config.h

```diff
@@ -50,4 +50,11 @@
 #define MBEDTLS_PLATFORM_ZEROIZE_ALT
 #define MBEDTLS_NO_DEFAULT_ENTROPY_SOURCES
 
+// This is only to pass a check in the mbedtls check_config.h header, none of
+// the files we include as part of the core build uses it anyway, we already
+// define MBEDTLS_PLATFORM_ZEROIZE_ALT which is the only relevant function.
+#if defined(__MINGW32__) || (defined(_MSC_VER) && _MSC_VER <= 1900)
```

Comment:
Since we don't support VS < 2017, I think you can remove the second part, even if it's matching what mbedtls checks. Otherwise we'll get a contributor cleaning up old unused defines in the future attempting to remove that one :P